app/views/collections/show.html.erb

```diff
@@ -3,12 +3,41 @@
   <div class="row">
     <%= render "layouts/sidebar" %>
     <main class="col-sm-9 col-md-10">
+      <style type="text/css">
+        body {
+          margin: 0;
+        }
+        #graph {
+          width: 400px;
+          height: 300px;
+          margin: auto;
+        }
+      </style>
       <h1><%= @collection.title %></h1>
       <p><%= button_to("Analyze Collection", user_collection_download_path(@user, @collection), data: {confirm: "Download and analyze " + collection_size(@collection.id, @user) + "?"}, class: 'btn btn-primary btn-lg btn-block') %></p>
       <%= link_to 'Back', current_user %>
       <% unless display_domains(@user.id, @collection.id, @collection.account).blank? %>
         <h2>Hyperlink Diagram</h2>
-        <p><%= display_gexf(@user.id, @collection.id, @collection.account) %></p>
+        <div id="graph"></div>
+        <%= javascript_tag do %>
```

Comment:
No worries!  The only minus is that the js must call $(document).ready() now.

Reply:
Just prompting a conversation on this review comment - for the "minus" you mentioned above, does that still work for you to implement Nick's suggestion, Ryan?

Reply:
$(document).ready() is common when running js from a separate file, so it's no worry.  SigmaJS recommends putting js on the page.  The main reason for this is that it would be a rare thing to want a network graph on every page of a site.  Otherwise this is fixed.

However, some js in the view is required to tell sigma what data to use because I need access to request-dependent data (page id, collection etc) from ruby.  See https://stackoverflow.com/questions/17809591/rails-3-1-insert-ruby-code-in-javascript-file

Reply:
OK thanks @greebie! - thoughts @ruebot?